src/SprykerEco/Service/Adyen/Dependency/Service/AdyenToUtilTextServiceBridge.php

```diff
@@ -23,12 +23,12 @@ public function __construct($textService)
     }
 
     /**
-     * @param string $algorithm
      * @param mixed $value
+     * @param string $algorithm
      *
      * @return string
      */
-    public function hashValue($algorithm, $value): string
+    public function hashValue($value, $algorithm): string
```

Comment:
This looks like a dangerous (and easily overlooked) change. Is this correct?

Reply:
\Spryker\Service\UtilText\UtilTextServiceInterface::hashValue